features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature

```diff
@@ -0,0 +1,67 @@
+@promotion
+Feature: Receiving fixed discount on products from specific taxon
+    In order to pay less while buying goods from promoted taxons
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store classifies its products as "T-Shirts" and "Mugs"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And it belongs to "Mugs"
+        And there is a promotion "T-Shirts promotion"
+        And it gives "€10.00" off every product classified as "T-Shirts"
+
+    @todo
+    Scenario: Receiving fixed discount on a single item from specific taxon
+        When I add product "PHP T-Shirt" to the cart
+        Then its price should be decreased by "€10.00"
+        And my cart total should be "€90.00"
+        And my discount should be "-€10.00"
+
+    @todo
+    Scenario: Receiving percentage discount on a multiple items from specific taxon
+        When I add 3 products "PHP T-Shirt" to the cart
+        Then their price should be decreased by "€10.00"
+        And my cart total should be "€270.00"
+        And my discount should be "-€20.00"
+
+    @todo
+    Scenario: Receiving fixed discount equal to the items total of my cart
+        Given there is a promotion "Christmas Sale"
+        And it gives "€20.00" off every product classified as "Mugs"
+        When I add 3 products "PHP Mug" to the cart
+        Then their price should be decreased by "€60.00"
+        And my cart total should be "€0.00"
+        And my discount should be "-€60.00"
+
+    @todo
+    Scenario: Receiving fixed discount equal to the items total of my cart even if the discount is bigger than the items total
```

Comment:
Invalid scenario, PHP mug price = €20, discount on every item = €20, cart total = €0, discount should be bigger than €20.